script.js

```diff
@@ -1,4 +1,165 @@
+// game mode
+var gameMode = "p1Input";
+var player1Number = null;
+var player2Number = null;
+var player1Rolls = [];
+var player2Rolls = [];
+var player1Numbers = [];
+var player2Numbers = [];
+var roundNumber = 1;
+var player1Sum = 0;
+var player2Sum = 0;
+
+
+var rollDice = function () {
+  // produces a float between 0 and 7
+  var randomFloat = Math.random() * 6;
+  // take off the decimal
+  var resultInteger = Math.floor(randomFloat) + 1;
+  lastRoll = resultInteger;
+  console.log("dice roll = " + resultInteger)
+  return resultInteger;
+  
+};
+
+var playerRoll = function() {
+  var roll1 = rollDice();
+  var roll2 = rollDice(); 
+  var rolls = [roll1, roll2];
+  console.log("roll numbers are " + rolls);
+  if (gameMode == "p1Input"){
+  gameMode = "player1Decision";
+  } else if (gameMode == "p2Input") {
+    gameMode = "player2Decision";
+  }
+  return rolls;
+}
+
+//calcualte the sum of elements in the array
+var playerSum = function(player) {
+  var counter = 0;
+  var sum = 0;
+  if(player == "player1"){
+    while (counter < roundNumber){
+      sum = sum + player1Numbers[counter];
+      counter ++;
+    }
+  } else if (player == "player2") {
+    while (counter < roundNumber){
+      sum = sum + player2Numbers[counter];
+      counter ++;
+     } 
+  }
+  return sum;
+}
+
+var playerDecideOrder = function(input, rolls, playerNumber) {
+  //var playerRolls = rolls;
```

Comment:
Remember to delete code that you're not using instead of just commenting it out.